core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -119,6 +119,19 @@ object ZIOSpec
             assertM(ZIO.fail("Fail").head.either, isLeft(isSome(equalTo("Fail"))))
           }
         ),
+        suite("ignore")(
+          testM("return success as Unit") {
+            import zio.CanFail.canFail
```

Comment:
These imports aren’t needed (that’s what makes the CI fail).

Reply:
Thanks 👍 I had those in from adapting the `.option` tests